CONTRIBUTING.md

```diff
@@ -0,0 +1,227 @@
+smoothly as possible and maximize the likelihood of your contribution being
+merged.
+
+How to contribute
+-----------------
+
+The preferred workflow for contributing to pydicom is to fork the
+[main repository](https://github.com/pydicom/pydicom) on
+GitHub, clone, and develop on a branch. Steps:
+
+1. Fork the [project repository](https://github.com/pydicom/pydicom)
+   by clicking on the 'Fork' button near the top right of the page. This creates
+   a copy of the code under your GitHub user account. For more details on
+   how to fork a repository see [this guide](https://help.github.com/articles/fork-a-repo/).
+
+2. Clone your fork of the pydicom repo from your GitHub account to your local disk:
+
+   ```bash
+   $ git clone git@github.com:YourLogin/pydicom.git
+   $ cd pydicom
+   ```
+
+3. Create a ``feature`` branch to hold your development changes:
+
+   ```bash
+   $ git checkout -b my-feature
+   ```
+
+   Always use a ``feature`` branch. It's good practice to never work on the ``master`` branch!
+
+4. Develop the feature on your feature branch. Add changed files using ``git add`` and then ``git commit`` files:
+
+   ```bash
+   $ git add modified_files
+   $ git commit
+   ```
+
+   to record your changes in Git, then push the changes to your GitHub account with:
+
+   ```bash
+   $ git push -u origin my-feature
+   ```
+
+5. Follow [these instructions](https://help.github.com/articles/creating-a-pull-request-from-a-fork)
+to create a pull request from your fork. This will send an email to the committers.
+
+(If any of the above seems like magic to you, please look up the
+[Git documentation](https://git-scm.com/documentation) on the web, or ask a friend or another contributor for help.)
+
+Pull Request Checklist
+----------------------
+
+We recommended that your contribution complies with the
+following rules before you submit a pull request:
+
+-  Follow the
+   [coding-guidelines](http://pydicom.org/dev/developers/contributing.html#coding-guidelines).
+
+-  Use, when applicable, the validation tools and scripts in the
+   `sklearn.utils` submodule.  A list of utility routines available
+   for developers can be found in the
+   [Utilities for Developers](http://pydicom.org/dev/developers/utilities.html#developers-utils)
+   page.
+
+-  If your pull request addresses an issue, please use the pull request title to
+   describe the issue and mention the issue number in the pull request
+   description. This will make sure a link back to the original issue is
+   created.
+
+-  All public methods should have informative docstrings with sample
+   usage presented as doctests when appropriate.
+
+-  Please prefix the title of your pull request with `[MRG]` (Ready for
+   Merge), if the contribution is complete and ready for a detailed review.
+   Two core developers will review your code and change the prefix of the pull
+   request to `[MRG + 1]` and `[MRG + 2]` on approval, making it eligible
```

Comment:
Merge+2 is always good policy. We use it in `imblearn` with a much smaller community and it works (most of the time).  It would reduce the load in @darcymason. But since all the integration runs through him this is already 1+1. So at this stage MRG+1 might be good enough to make it eligible.